static/lato.css

```diff
@@ -0,0 +1,89 @@
+/* Adapted from https://fonts.googleapis.com/css?family=Lato:400,400i,700,900 */
+
+/* non-latin (cyrillic, hungarian, serbian) */
+@font-face {
+    font-family: 'Lato';
+    font-style: italic;
+    font-weight: 400;
+    src: local('Lato Italic'), local('Lato-Italic'), url(fonts/lato/Lato-Italic.woff2) format('woff2');
+}
+
+@font-face {
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Lato Regular'), local('Lato-Regular'), url(fonts/lato/Lato-Regular.woff2) format('woff2');
+}
+
+@font-face {
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 700;
+    src: local('Lato Bold'), local('Lato-Bold'), url(fonts/lato/Lato-Bold.woff2) format('woff2');
+}
+
+@font-face {
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 900;
+    src: local('Lato Black'), local('Lato-Black'), url(fonts/lato/Lato-Black.woff2) format('woff2');
+}
+
+/* latin */
+@font-face {
+    font-family: 'Lato';
```

Comment:
Since this font-family is the same as the one above, and doesn't have any unicode ranges, won't that mean that we end up overriding the regular one with the latin one?

Reply:
Good point, I'll add unicode ranges to the ones above (cyrillic).

Reply:
@jeresig I'm going to implement these changes in parallel on webapp so I'll make sure to pass the correct fonts and unicodes. I'll update this PR in case I found something else missing.

Reply:
Ok, sounds good!

Reply:
Ugh.. so far these are the results of webapp using the new fonts:

### LOHP Overall font size

Without new fonts: 167 KB
New fonts included (current PR): 393 KB

I've noticed that the website is loading the LatinExtended font (where it shouldn't be). I'll have to make some teaks and will come back here after getting better results.